langchain4j-bedrock/src/test/java/dev/langchain4j/model/bedrock/BedrockChatModelListenerIT.java

```diff
@@ -0,0 +1,73 @@
+package dev.langchain4j.model.bedrock;
+
+import dev.langchain4j.model.chat.ChatLanguageModel;
+import dev.langchain4j.model.chat.ChatModelListenerIT;
+import dev.langchain4j.model.chat.listener.ChatModelListener;
+import software.amazon.awssdk.regions.Region;
+
+import java.time.Duration;
+import java.util.Collections;
+
+public class BedrockChatModelListenerIT extends ChatModelListenerIT {
+    @Override
+    protected ChatLanguageModel createModel(ChatModelListener listener) {
+        return BedrockAnthropicMessageChatModel
+                .builder()
+                .temperature(0.50f)
```

Comment:
no need to override the value for temperature and other parameters, please check how other similar tests are implemented